tensorflow/compiler/aot/tfcompile_main.cc

```diff
@@ -136,6 +136,10 @@ int main(int argc, char** argv) {
 
   tensorflow::string usage = tensorflow::tfcompile::kUsageHeader;
   usage += tensorflow::Flags::Usage(argv[0], flag_list);
+  if (argc > 1 && !strcmp(argv[1], "--help")) {
+    std::cerr << usage << "\n\n";
```

Comment:
What's the purpose of the `\n\n`?  I'd have expected at most one `\n` would be necessary.

Reply:
Thanks for review, I was just watching to line https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/aot/tfcompile_main.cc#L144
I'll fixed it.